scripts/.tests.env

```diff
@@ -10,7 +10,7 @@ export ANDROID_SDK_BUILD_API_LEVEL="28"
 # Google APIs for Android level
 export ANDROID_GOOGLE_API_LEVEL="23"
 # Minimum Android API Level we target
-export ANDROID_SDK_TARGET_API_LEVEL="19"
+export ANDROID_SDK_TARGET_API_LEVEL="21"
```

Comment:
This can be set to 29. Google is forcing it https://developer.android.com/distribute/best-practices/develop/target-sdk.

Reply:
From ReactAndroid/build.gradle:

```
        minSdkVersion(21)
        targetSdkVersion(28)
```

So if the comment is right, then 21 is correct, unless the comment is wrong:

> # Minimum Android API Level we target

Reply:
Sorry, judge from the name, I think it's targetSdkVersion.